README.md

```diff
@@ -2,42 +2,98 @@ PReview Code frontend [![Build Status](https://travis-ci.org/preview-code/fronte
 =========
 This is the frontend code for https://preview-code.com implemented with Polymer.
 
-# Installation for testing
+Please read the complete README.md file before executing any of the steps.
 
-Make sure you have `web-component-tester` installed globally as well as
-the plugin `web-component-tester-istanbul`:
+If you live in the Netherlands, please consider using the following mirror in "Ubuntu Software & Updates":
+<ftp://ftpserv.tudelft.nl/pub/Linux/archive.ubuntu.com>
 
-```
-sudo npm i -g web-component-tester web-component-tester-istanbul
-```
+# Installation guide
+## Prerequisites: nodejs, npm, and git
+1. Install the LTS version (4.x) of [node.js](https://nodejs.org/en/). Please __do not__ install version 6.0 as it is not officially supported by [polymer](https://www.polymer-project.org/1.0/start/first-element/intro) (last checked on 2016.09.22).
+ 
+		curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
+		sudo apt-get install nodejs -y
+2. install the _latest_ version of [npm](https://www.npmjs.com/):
 
-# Installation for serviceWorker
-Service-worker requires you to run the website on `https`. Therefore you need
-to install a local self-signed certificate for `localhost:5200`.
+		sudo npm install npm@latest -g
+3. install [git](https://git-scm.com/):
```

Comment:
Indeed it is. I included git because earlier I had also included Atom (now removed) which depends on git. However, a later step clones the frontend, and if we start from clean installation then some sort of means must be available to produce the clone.